### PR TITLE
Vue3 apollo quasar/test/pagination buttons component

### LINKED ISSUE #478: [vue3-apollo-quasar] unit test for PaginationButtons component
The expectation of this ticket is to write the test properly using best practices.
Currently the wrapper is declared in a global state which shouldn't be.
[read more on this topic](https://jestjs.io/docs/setup-teardown)
Also it's not good practice having if statement in test. To fix that, you can [read more here](https://dev.to/bgord/simplify-repetitive-jest-test-cases-with-test-each-310m)

### PR DESCRIPTION
**Pull request by hdJerry**
closes #478 

Test completed and re-written

### CODE REVIEW

File: vue3-apollo-quasar/src/components/PaginationButtons/PaginationButtons.vue [hunk 1]
```diff
@@ -1,15 +1,15 @@
 <template>
   <div class="row justify-center items-center q-my-md">
     <button
```
Comment (AllanJeremy) [inline vue3-apollo-quasar/src/components/PaginationButtons/PaginationButtons.vue:3]: We had discussed previously that once we got to the pagination, we'd use the quasar component.

Got mulltiple concerns with this: 
- Using HTML `button` component despite the discussion to prefer quasar components
- Creating this component altogether since quasar already has[ a prebuilt pagination component](https://quasar.dev/vue-components/pagination) that would work much better than trying to reinvent the will

## TL;DR
Kindly use the [quasar pagination components](https://quasar.dev/vue-components/pagination) for this and customize styling if need-be.

File: vue3-apollo-quasar/tests/unit/components/PaginationButtons.spec.ts [hunk 2]
```diff
@@ -1,33 +1,80 @@
...
+  wrapper = mount(PaginationButtons, {
     props: {
       isPrevActive: false,
       isNextActive: true,
     },
   });
-
-  const pagination_btns = wrapper.findAll('.pagination_btn');
+});
+describe('PaginationButtons mounting', () => {
```
Comment (AllanJeremy) [inline vue3-apollo-quasar/tests/unit/components/PaginationButtons.spec.ts:14]: Tests look good. Please check out the previous comment though

**Review by AllanJeremy — CHANGES_REQUESTED** [review]

File: vue3-apollo-quasar/tests/unit/components/PaginationButtons.spec.ts (hunk 2 above)
Comment (AllanJeremy) [reply vue3-apollo-quasar/tests/unit/components/PaginationButtons.spec.ts:14]: @hdJerry , I understand that this is the pagination PR but if we keep pushing to do it (switch to quasar component) later (as we did previously) it'll never happen

File: vue3-apollo-quasar/src/components/PaginationButtons/PaginationButtons.vue (hunk 1 above)
Comment (AllanJeremy) [reply vue3-apollo-quasar/src/components/PaginationButtons/PaginationButtons.vue:3]: @hdJerry

File: vue3-apollo-quasar/tests/unit/components/PaginationButtons.spec.ts [hunk 3]
```diff
@@ -1,33 +1,11 @@
 import { mount } from '@vue/test-utils';
 import { PaginationButtons } from '@/components';
```
Comment (AllanJeremy) [inline vue3-apollo-quasar/tests/unit/components/PaginationButtons.spec.ts:3]: Please add tests that test for the actual functionality:
- Displays pagination buttons
- Tests that clicking pagination works
- Tests that next and previous buttons work


Testing for mount functionality only doesn't provide any value

**Review by AllanJeremy — CHANGES_REQUESTED** [review]

File: vue3-apollo-quasar/tests/unit/components/PaginationButtons.spec.ts (hunk 3 above)
Comment (hdJerry) [reply vue3-apollo-quasar/tests/unit/components/PaginationButtons.spec.ts:3]: Can't do deep testing.

**Review by AllanJeremy — APPROVED** [review]
:shipit:
